README.md

```diff
@@ -12,24 +12,23 @@ What's Piri? It's the _**P**rovable **I**nformation **R**etention **I**nterface_
 
 Get started with Piri by exploring our comprehensive documentation:
 
-- **[🚀 Getting Started](./docs/getting-started.md)** - Complete setup guide to deploy Piri
+- **[📖 Start Here](./docs/content/index.md)** - Pick staging vs. production and jump to the right getting started guide
```

Comment:
This is still the getting started guide...I think I preferred the previous text...